src/Compilers/Core/Portable/StrongName/DesktopStrongNameProvider.cs

```diff
@@ -118,11 +120,14 @@ internal sealed class ClrStrongNameMissingException : Exception
 
         private readonly ImmutableArray<string> _keyFileSearchPaths;
         private readonly string _tempPath;
+        internal StrongNameFileSystem FileSystem { get; } = new StrongNameFileSystem();
```

Comment:
This is allocated twice in case where `.ctor` argument is `null`.

Reply:
Done.